porting/nimble/src/os_mbuf.c

```diff
@@ -633,19 +626,20 @@ os_mbuf_adj(struct os_mbuf *mp, int req_len)
          * after the remaining size.
          */
         len = -len;
+        if (m->om_len >= len) {
+            m->om_len -= len;
+            if (OS_MBUF_IS_PKTHDR(mp))
+                OS_MBUF_PKTHDR(mp)->omp_len -= len;
+            return;
+        }
```

Comment:
this will trim the length of 1st mbuf in chain instead of the last one

Reply:
That's right. My bad. Closing the PR.